### DISCUSSION TITLE
SPC SPC for C-c *and* SPC SPC SPC for h-SPC

### DISCUSSION THREAD

**Discussion by eeriemyxi** [meow-edit/meow#673] [Q&A] [answered]
Can I bind `SPC SPC` to `C-c` then bind `SPC SPC SPC` to previous function of `SPC SPC`? I ended up with recursion with what I tried.

**Comment by DogLooksGood** [comment]
I don't quite understand your question, if `SPC SPC` is `C-c`(assume you have a command on `C-c`), then binding `SPC SPC SPC` is impossible.

**Reply by eeriemyxi** [reply]
If I bind `SPC SPC` to `C-c` then bind `C-c SPC` to something that makes it act like the previous behavior of `SPC SPC` I think it might work. But I don't know what that _something_ is.

**Reply by DogLooksGood** [reply]
AFAIK, there's no *something* that represents the previous behavior.

**Comment by eeriemyxi** [comment, marked as answer]
```lisp
(meow-leader-define-key
 '("SPC"         . "C-c C-SPC")
 '("C-SPC SPC"   . "H-SPC"))
```
Use `C-c C-SPC` instead of just `C-c`. You get to keep defaults of most major mode packages that bind their commands to `C-c` while enjoying your own shortcuts via the shorthand `SPC SPC` in normal or motion mode.
